src/constants.js

```diff
@@ -41,4 +41,7 @@ export const PROGRAMME_PATH = '/utdanning';
 export const PROGRAMME_PAGE_PATH = '/utdanning/:programme';
 
 export const MULTIDISCIPLINARY_SUBJECT_PAGE_PATH =
-  '/subject:d1fe9d0a-a54d-49db-a4c2-fd5463a7c9e7';
+  '/subject:d1fe9d0a-a54d-49db-a4c2-fd5463a7c9e7/:topicPath*/';
+
+export const MULTIDISCIPLINARY_SUBJECT_ARTICLE_PAGE_PATH =
+  '/subject:d1fe9d0a-a54d-49db-a4c2-fd5463a7c9e7/:topicPath*/card';
```

Comment:
Korleis blir det med ressurser under dette emnet. Dei blir kanskje vist som tidligere?

Reply:
Jeg har ikke endret på article-siden (annet enn å flytte det til `.../card`, så regner med at det vises som tidligere, men jeg kan sjekke det ut.

Reply:
Testa det og det så ut til å fungere fint (Etter jeg skjønte at taksonomi ressurser var usynelige by default og måtte ha filter tilknytning for å vises)

Reply:
Lenkene på forsida skal oppdateres til å peke på dei tre topp-emnene, og uten filter.